docs/release-notes/deprecate-job-move.rst

```diff
@@ -0,0 +1,7 @@
+:orphan:
+
+**Deprecations**
+
+-  Agent and Kubernetes Resource Manager: Jobs can no longer be moved within the same priority
+   group. Update a job's priority via CLI command to move a job ahead, or behind, of another job.
+   Visit :ref:`modify-job-queue-cli` for details. Deprecation was announced in 0.33.0.
```

Comment:
- Agent and Kubernetes Resource Manager: Jobs can no longer be moved within the same priority group. To reposition a job, update its priority using the CLI. For detailed instructions, visit :ref:`modify-job-queue-cli`. This change was announced in version 0.33.0.

Reply:
note to self: adjust priority in CLI or webui